rmap-project/rmap
# Add button to keys view to copy the key to clipboard

This change adds an additional option to the `/keys/` display to copy the key to the clipboard. Previously, the Key table displayed "edit | download" as options, now you can edit by clicking the `label` or `key id` and the buttons offered are `copy key` and `download key`.

This change also enhances the instructional text on the keys management page and the keys form.

To test:
1. Get a copy of this branch 
2. Follow [instructions for running RMap locally](https://github.com/rmap-project/rmap/blob/master/DEVELOPER.md#running-rmap)
3. Login to the admin area using `http://localhost:{port#}/app/admin/login` with user name rmapAdmin/rmapAdmin
4. Go to User management page, add a user and add a key for that user.
5. Visit the keys page for the new user and use the button to copy the key to clipboard
6. Paste the key to make sure it is in the clipboard

Note you can also test this by going to [https://test.rmap-hub.org](https://test.rmap-hub.org), creating and account using Google or Twitter, creating a key and testing the button that way.
